### DISCUSSION TITLE
cursor looks too small

### DISCUSSION THREAD

**Discussion by ghost** [antiden/macOS-cursors-for-Windows#60] [Q&A] [answered]
ugh

**Comment by antiden** [comment]
Hi @notrealkittenstrike ,
What is the screen resolution?

**Reply by ghost** [reply]
the same as the size of a dell vostro 15

**Reply by antiden** [reply]
Please type your screen resolution.

**Reply by ghost** [reply]
1920x1080 px

**Reply by antiden** [reply, marked as answer]
This is cursors ONLY for "Tested in Windows 10 & 11, 4K" - 3840 x 2160 pixel (4K UHDTV)

**Reply by ghost** [reply]
ok, thanks. guess ill use another cursor
btw i love ur cursors
